src/components/Header/Header.spec.tsx

```diff
@@ -37,12 +39,36 @@ test('Displays the header', () => {
   const link = screen.getByRole('link', { name: 'Menu' })
   expect(link).toHaveAttribute('href', '/browse')
   expect(link).toHaveClass('js-disabled-show')
+
+  // Back link
+  expect(screen.getByRole('link', { name: 'Back to list of data service providers' })).toHaveAttribute(
```

Comment:
`getByRole`, `queryByRole` etc can be destructured from the render method, ie

`const { getByRole, queryByRole } = render(<Header heading="Empty Heading" isPreviewMode={false} />)`

Reply:
updated